tutorial/tutorial.thrift

```diff
@@ -113,7 +113,7 @@ struct Work {
  * Structs can also be exceptions, if they are nasty.
  */
 exception InvalidOperation {
-  1: i32 what,
+  1: i32 whatOp,
```

Comment:
"what" needs to be added to the compiler's list of keywords as part of this work.